examples/cra-kitchen-sink/src/stories/index.stories.js

```diff
@@ -236,70 +158,3 @@ storiesOf('WithEvents', module)
     </WithEvents>
   ))
   .add('Logger', () => <Logger emiter={emiter} />);
-
-storiesOf('withNotes', module)
-  .add('with some text', withNotes('Hello guys')(() => <div>Hello guys</div>))
-  .add('with some emoji', withNotes('My notes on emojies')(() => <p>🤔😳😯😮</p>))
-  .add(
-    'with a button and some emoji',
-    withNotes('My notes on a button with emojies')(() => (
-      <Button onClick={action('clicked')}>😀 😎 👍 💯</Button>
-    ))
-  )
-  .add('with old API', () => (
-    <WithNotes notes="Hello">
-      <Button onClick={action('clicked')}>😀 😎 👍 💯</Button>
-    </WithNotes>
-  ));
-
-storiesOf('component.base.Link', module)
```

Comment:
I got rid of these because they don't seem to be testing or documenting anything in particular. Seems like a waste of space and code to me.

Reply:
I think they are for testing deep nesting of stories

Reply:
I think in this case it should be okay to delete. The examples for Addon Info are pretty well nested too.

Reply:
Yeah, it was added for the hierarchy feature. If other examples are descriptive enough to demonstrate the hierarchy, it's ok to delete all of these.